tests/unit_tests/test_engine.py

```
@@ -182,3 +183,11 @@ def sample_login_method():
 
     assert response.status_code == 200
     assert b"Login Method" in response.data
+
+
+def test_add_cms_module(test_app):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)